### DISCUSSION TITLE
Image warp

### DISCUSSION THREAD

**Discussion by Neobii** [fabricjs/fabric.js#6882] [Q&A]
Was looking through some packages for some tooling to work with the canvas.  Do ya'll have any plans to add an image warp like [this](https://github.com/cxcxcxcx/imgwarp-js/)?

**Comment by asturur** [comment]
I have an example of that in fabricJS but is not something that fits in the library. Is something you can integrate in some way between the control api and either a image filter ( i did in that way ) or a subclass.
The issue here is that then bounding box is never looking good.
